lib/m2ts/m2ts.js

```diff
@@ -269,6 +317,84 @@ ElementaryStream = function() {
       data: [],
       size: 0
     },
+    handlePesData = function(data) {
```

Comment:
You could make these private-ish methods and then skip the need for `self` (e.g. `handlePesData_` or just `pes_`).